slave.mk

```diff
@@ -630,15 +635,51 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		export docker_container_name="$($(docker:-dbg.gz=.gz)_CONTAINER_NAME)"
 		$(eval $(docker:-dbg.gz=.gz)_RUN_OPT += $($(docker:-dbg.gz=.gz)_$($*_IMAGE_TYPE)_RUN_OPT))
 		export docker_image_run_opt="$($(docker:-dbg.gz=.gz)_RUN_OPT)"
-		j2 files/build_templates/docker_image_ctl.j2 > $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).sh
+		if [ "$(MULTI_ASIC)" = "y" ]; then
+			export multi_asic="True"
+		else
+			export multi_asic="False"
+		fi
+
 		if [ -f files/build_templates/$($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service.j2 ]; then
 			j2 files/build_templates/$($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service.j2 > $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service
 		fi
+
+		if [ "$(MULTI_ASIC)" = "y" ]; then
+			export multi_instance="True"
```

Comment:
what is the difference between
multi_asic vs multi_instance

Reply:
multi_asic indicates that we are building an image for a device with multiple ASICs. Multi_instance means we need multiple copies of a particular container